### DISCUSSION TITLE
Eradicate Hacker News "New" section

### DISCUSSION THREAD

**Discussion by arximboldi** [jordwest/news-feed-eradicator#423] [Ideas]
My brain has figured out that it can click on Hacker News "new" button to escape the eradicator. Maybe the eradicator could eradicate that feed as well?

Thank you for this amazing tool. This and uBlock are my two indispensable browser extensions. I love this.

**Comment by kimjune01** [comment]
you can actually do this with ublock. right click on the "new" button and block element.

**Comment by arximboldi** [comment]
It's a great tip! Thanks! I will use it :smile:
